lib/pages/V1App.dart

```diff
@@ -395,6 +398,17 @@ class _V1AppState extends State<V1App> {
                         size: 25,
                         color: Colors.grey[400],
                       )),
+                  Container(width: _width - 320),
+                  SimpleToolTip(
+                      "Invest in what you know best at AthleteX Markets. Click here to learn more!",
+                      IconButton(
+                          onPressed: () =>
+                              launchUrl(Uri.parse('https://www.athletex.io/')),
```

Comment:
lets change the link to https://athletex-markets.gitbook.io/athletex-huddle/start-here/litepaper